### ISSUE TITLE
Error: ENOENT: no such file or directory, lstat

### ISSUE THREAD

**Issue by trusktr** [mwittig/npm-license-crawler#28]
I tried the example from the README, and it resulted in an error on macOS Mojave:

```
❯ npm-license-crawler  --exclude ./lib/logging --dependencies --csv licenses.csv
{ Error: ENOENT: no such file or directory, lstat '/Users/trusktr/src/project/lib/logging'
    at Object.lstatSync (fs.js:861:3)
    at /Users/trusktr/.npm-packages/lib/node_modules/npm-license-crawler/lib/directoryreader.js:27:41
    at processTicksAndRejections (internal/process/next_tick.js:74:9)
    at process.runNextTicks [as _tickCallback] (internal/process/next_tick.js:51:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:804:11)
    at internal/main/run_main_module.js:21:11
  errno: -2,
  syscall: 'lstat',
  code: 'ENOENT',
  path:
   '/Users/trusktr/src/project/lib/logging' }
```

**Comment by mwittig** [comment]
Hi, there. Thanks for trying npm-license-crawler! 

Make sure, the given exclude path is valid. If you run `npm-license-crawler` from the `npm-license-crawler` directory the following command line example works for me:
`npm-license-crawler  --exclude ./lib/logging --dependencies --csv licenses.csv`

Take-aways for me:
* Improve error handling for exclude path check
* Revise README to give a working example

**Comment by mwittig** [comment]
published npm-license-crawler@0.2.0
